common/changes/@itwin/core-quantity/rob-almost-equal-fix_2024-09-06-07-05.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-quantity",
+      "comment": "New almostEqual method for comparing numbers",
```

this is an internal api, not exposed to the user, therefore no need to mention it in the changelog

```suggestion
      "comment": "",
```